### ISSUE TITLE
Add Zoom in/out buttons to workspace in Tutorial Mode

### ISSUE THREAD

**Issue by Jaqster** [Microsoft/pxt#4885]
Accessibility issue - when doing a Tutorial, users are not able to zoom in/out.  This can be a problem when user screens are small - they cannot see all their blocks in the default view.
